### ISSUE TITLE
AirMAX Quality/Capacity monitoring

### ISSUE THREAD

**Issue by jonasrita99** [Ubiquiti-App/UNMS#141]
It would be very nice if UNMS implement a way to check AirMAX Quality and Capacity and create alarms if these valors get too much low.

Is that any plan to implement this?

**Comment by radekskrivan** [comment]
@jonasrita99   Yes, we have a plan to improve upon our current alert system and this feature is a part of it. We would like to have this in 0.13.0 release.
